services/ansible/ansible-role.service.js

```diff
@@ -47,15 +48,14 @@ class AnsibleGalaxyRoleName extends AnsibleGalaxyRole {
   static category = 'other'
   static route = { base: 'ansible/role', pattern: ':roleId' }
 
-  static examples = [
-    {
-      title: 'Ansible Role',
-      namedParams: { roleId: '3078' },
-      staticPreview: this.render({
-        name: 'ansible-roles.sublimetext3_packagecontrol',
-      }),
+  static openApi = {
+    '/ansible/role/d/{roleId}': {
```

Comment:
Is this right? Based on the route I wouldn't have expected the `/d/` portion here